apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/TransactionPool.cs

```diff
@@ -243,40 +245,47 @@ public void MarkTransactionUsed()
                 }
             }
 
-            public async Task CreateTransactionAsync(Session session, TransactionOptions options)
+            public Task CreateTransactionAsync(Session session, TransactionOptions options)
```

Comment:
Presumably this could be called from multiple threads simultaneously. Is that okay? I've lost track of the threading model here...